cogs/r5rs.scm

```diff
@@ -386,12 +386,14 @@
 (check-equal? "string length correctly reported for standard string" 3 (string-length "abc"))
 
 (check-equal? "string indexing into first character" #\a (string-ref "abc" 0))
+(check-equal? "string indexing with multibyte characters" #\a (string-ref "λa" 1))
```

Comment:
I could add way more tests, but I'm not sure if this is the right place.

Reply:
you can add tests here, or if you'd like more direct ones, you can make a file in either the `success` or `failure` directories [here](https://github.com/mattwparas/steel/tree/master/crates/steel-core/src/tests)

And then add the file name in the requisite handler [here](https://github.com/mattwparas/steel/blob/master/crates/steel-core/src/tests/mod.rs#L52)

Reply:
I added just a couple more.